UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/FileContentHelper.cs

```diff
@@ -366,5 +345,42 @@ public static void CheckSerialEncFileContentForLargeMediaExchangeSet(string inpu
             Assert.AreEqual(dateAndCdType, $"{currentDate}BASE", $"Incorrect date is returned '{currentDate}UPDATE', instead of the expected {dateAndCdType}.");
             Assert.IsTrue(formatVersionAndExchangeSetNumber.StartsWith($"02.00B0{baseNumber}X09"), $"Expected format version {formatVersionAndExchangeSetNumber}");
         }
+
+        public static async Task<List<string>> ExchangeSetLargeFile(HttpResponseMessage apiEssResponse, string FssJwtToken)
+        {
+            List<string> downloadFolderPath = new List<string>();
+            for (int mediaNumber = 1; mediaNumber <= 2; mediaNumber++)
+            {
+                var FolderName = $"M0{mediaNumber}X02";
+                Assert.AreEqual(200, (int)apiEssResponse.StatusCode, $"Incorrect status code is returned {apiEssResponse.StatusCode}, instead of the expected status 200.");
+
+                var finalBatchStatusUrl = $"{Config.FssConfig.BaseUrl}/batch/62713adc-6999-40f6-86b1-ca08ab693d44/status"; //here BatchId is hardcoded and will be made dynamic in future
+
+                var batchStatus = await FssBatchHelper.CheckBatchIsCommitted(finalBatchStatusUrl, FssJwtToken);
+                Assert.AreEqual("Committed", batchStatus, $"Incorrect batch status is returned {batchStatus} for url {finalBatchStatusUrl}, instead of the expected status Committed.");
+
+                var downloadFileUrl = $"{Config.FssConfig.BaseUrl}/batch/62713adc-6999-40f6-86b1-ca08ab693d44/files/{FolderName}.zip"; //here BatchId is hardcoded and will be made dynamic in future
+
+                var extractDownloadedFolder = await FssBatchHelper.ExtractDownloadedFolderForLargeFiles(downloadFileUrl.ToString(), FssJwtToken, FolderName);
```

Comment:
downloadFileUrl is a string, so the .ToString() is redundant and can be removed.

Reply:
Updated.

Thanks